app/src/org/commcare/dalvik/activities/EntitySelectActivity.java

```diff
@@ -901,6 +918,12 @@ public void onClick(View v) {
     }
 
     @Override
+    public void onSaveInstanceState(Bundle savedInstanceState) {
+        savedInstanceState.putBoolean(CONTAINS_HERE_FUNCTION, containsHereFunction);
+        super.onSaveInstanceState(savedInstanceState);

```

Comment:
Stylistically, I prefer to see calls to `super.currentMethod` come first when ordering isn't important, such as this case.